test/e2e/e2e.go

```diff
@@ -815,7 +819,17 @@ func deleteMetalLB(metallb *metallbv1beta1.MetalLB) {
 	}, timeout, interval).Should(BeTrue())
 }
 
-var _ = BeforeSuite(func() {
-	_, err := testclient.Client.Namespaces().Get(context.Background(), OperatorNameSpace, metav1.GetOptions{})
-	Expect(err).ToNot(HaveOccurred(), "Should have the MetalLB Operator namespace")
-})
+func getMetalLB() (*metallbv1beta1.MetalLB, error) {
+	metallb := &metallbv1beta1.MetalLB{}
+	if UseMetallbResourcesFromFile {
+		err := loadMetalLBFromFile(metallb, consts.MetalLBCRFile)
+		if err != nil {
+			return nil, err
+		}
+		return metallb, nil
+	} else {
+		metallb.SetName("metallb")
+		metallb.SetNamespace(OperatorNameSpace)
```

Comment:
we need to set the namespace when we load metallb from file as well

Reply:
fixed